### PR TITLE
feat: add active page in navbar

### LINKED ISSUE #82: feat: active page in navbar
- For example, `Profile` text in navbar becomes indigo if the user is on that page
- Use next/router (pathname)

### PR DESCRIPTION
**Pull request by ssarkar551**

### CODE REVIEW

File: src/components/Misc/Navbar.tsx [hunk 1]
```diff
@@ -25,6 +27,11 @@ export const Navbar = () => {
           <NavigationMenu.Link
             href="/"
             className={clsx(
+              `${
+                router.pathname === "/"
+                  ? "dark:indigo-300 text-indigo-700"
```
Comment (AyanavaKarmakar) [inline src/components/Misc/Navbar.tsx]:
```suggestion
                  ? "dark:text-indigo-200 text-indigo-700"
```

File: src/components/Misc/Navbar.tsx [hunk 2]
```diff
@@ -4,9 +4,11 @@ import { LoginButton, LogoutAndAlertButton, ThemeToggle } from "..";
 import { regularFont } from "../../fonts";
 import clsx from "clsx";
 import { motion } from "framer-motion";
+import { useRouter } from "next/router";
 
 export const Navbar = () => {
   const { status } = useSession();
+  const router = useRouter();
```
Comment (AyanavaKarmakar) [inline src/components/Misc/Navbar.tsx]:
```suggestion
  const { pathname } = useRouter();
```

Comment (AyanavaKarmakar) [reply src/components/Misc/Navbar.tsx]: only grab the required prop, the entire router instance is not needed

File: src/components/Misc/Navbar.tsx (hunk 1 above)
Comment (AyanavaKarmakar) [reply src/components/Misc/Navbar.tsx]: fix for other routes too

**Review by AyanavaKarmakar — CHANGES_REQUESTED** [review]

**Review by AyanavaKarmakar — APPROVED** [review]

**Review by Nishith-Savla — APPROVED** [review]

**Pull request merged**